### PR TITLE
iron-component-page polish of the day

### PR DESCRIPTION
**Pull request by aomarks**
Live at https://aomarks-test-docs.appspot.com/app-layout.html#/elements/app-header-layout

### CODE REVIEW

File: iron-component-page.html [hunk 1]
```diff
@@ -51,8 +52,9 @@
 
       app-drawer {
         --app-drawer-content-container: {
-          background-color: #f9f9f9;
+          background-color: #fbfbfb;
         }
+        box-shadow: 0 0 8px rgba(0, 0, 0, 0.3)
```
Comment (samuelli) [inline iron-component-page.html]: ;

**Comment by justinfagnani** [comment]
I'd still like to see some indication that the expandable nav section is demos. What do you think about API and Demos subheaders?

File: iron-component-page.html (hunk 1 above)
Comment (aomarks) [inline iron-component-page.html]: Done.

**Comment by aomarks** [comment]
@justinfagnani Yeah, I'll play around with that for next PR.

**Pull request merged**